src/messaging.js

```diff
@@ -0,0 +1,58 @@
+import {lmsAddress} from './config';
```

Comment:
@rodrigopavezi As discussed, I think we should make message topics lowercase to fit what the modules are doing across the board even though on the Twitter module-end any case will work. Thoughts?

Reply:
@jungeunyoon sounds good. I will change to lower case. cheers